packages/braid-design-system/src/lib/components/Toggle/Toggle.docs.tsx

```diff
@@ -58,6 +58,34 @@ const docs: ComponentDocs = {
           </Stack>,
         ),
     },
+    {
+      label: 'Toggle position',
+      description: (
+        <Text>
+          Toggles can be positioned before or after the label via the{' '}
+          <Strong>togglePosition</Strong> prop.
```

Comment:
As above, it would be good to reword to incorporate the names of the values here. 

Maybe something like this:

> By default, the position of the toggle relative to the label text will be determined by its `<Strong>align</Strong>` prop. This can be overridden by setting the `<Strong>togglePosition</Strong` prop as either `<Strong>leading</Strong>` or `<Strong>trailing</Strong>`

Thoughts?